### PR TITLE
{chore} Add support fro Angular 18-rc and bump playwright/test version

### PR DESCRIPTION
**Pull request by waztsup**
Add support for Angular 18-rc.
Bump playwright/test because of [this](https://github.com/microsoft/playwright/issues/15717) issue.

EDIT by @bgotink to stop gh from attaching this PR as "possible fix" to the linked playwright ticket

### CODE REVIEW

File: packages/harness/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		"./package.json": "./package.json"
 	},
 	"peerDependencies": {
-		"@angular/cdk": "^13.3.2 || ^14.0.0-rc.0 || ^15.0.0-rc.0 || ^16.0.0-rc.0 || ^17.0.0"
+		"@angular/cdk": ">=13.3.2"
```
Comment (jVerhaert) [inline packages/harness/package.json:13]: Thanks for changing it to a minimal version, which makes it little more friendly for next upcoming Angular versions, but the change isn't applied in the lock file yet.

Comment (waztsup) [reply packages/harness/package.json:13]: Ops, forgot to run `yarn`, should be good now.

File: yarn.lock [hunk 2]
```diff
@@ -2767,14 +2767,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@playwright/test@npm:^1.20.2":
-  version: 1.41.1
-  resolution: "@playwright/test@npm:1.41.1"
+"@playwright/test@npm:^1.41.2":
```
Comment (bgotink) [inline yarn.lock:2770]: I'm not sure why this update was necessary, could you explain?

The linked issue relates to a change fixed in 1.24 and this repo was already using 1.41.
Updating the version in the package.json forces others to update, which would make this PR a bigger change than it needs to be.

File: packages/harness/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
 		"./package.json": "./package.json"
 	},
 	"peerDependencies": {
-		"@angular/cdk": "^13.3.2 || ^14.0.0-rc.0 || ^15.0.0-rc.0 || ^16.0.0-rc.0 || ^17.0.0"
```
Comment (bgotink) [inline packages/harness/package.json:13]: I believe I already told @jVerhaert via chat, but I don't want to use a `>= X` range. It's impossible to predict forward compatibility with `@angular/cdk`, so this range would be a lie.
(See also the `>= 8.43.0` eslint range issue we had earlier this year when eslint 9 came out)

**Review by bgotink — CHANGES_REQUESTED** [review]

**Comment by bgotink** [comment]
Angular 18 support is now available in `@ngx-playwright/test@0.5.6`

**Pull request closed without merging**